php/blocks/class-loader.php

```diff
@@ -569,15 +569,15 @@ public function add_block( $block_config ) {
 	 * This method should be called during the block_lab_add_blocks action, to ensure
 	 * that the block isn't added too late.
 	 *
-	 * @param array $block_name The name of the block that the field is added to.
+	 * @param string $block_name The name of the block that the field is added to.
 	 * @param array $field_config The config of the field to add.
 	 */
 	public function add_field( $block_name, $field_config ) {
 		if ( ! isset( $this->blocks[ "block-lab/{$block_name}" ] ) ) {
 			return;
 		}
 		if ( ! isset( $field_config['name'] ) ) {
-			return $blocks;
+			return;
```

Comment:
Is it alright that I made this change? Maybe something else was intended.

Reply:
No, good pickup.